config/sys.config

```diff
@@ -11,6 +11,9 @@
 			{request_labels, [http_method, route, reason, status_class]},
 			{error_labels, [http_method, route, reason, error]},
 			{labels_module, ar_prometheus_cowboy_labels}
-		]}
+		]},
+		{vm_system_info_collector_metrics, []},
```

Comment:
Omit the following metrics:
- `erlang_vm_msacc_XXX`
- `erlang_vm_allocators`
- `erlang_vm_dist_XXX`

Together they represent about 2,000 individual metrics that aren't widely used (if used at all)